Network/Socket/Types.hsc

```diff
@@ -162,6 +163,18 @@ withFdSocket (Socket ref _) f = do
   touch ref
   return r
 
+-- | Socket is closed and a duplicated file descriptor is returned.
```

Comment:
The documentation can they say, that the duplicated descriptor is no longer subject to the possibility of unexpectedly being closed if the socket is finalized. It is now the caller's responsibility to ultimately close the duplicated file descriptor.

Reply:
Thanks and done.